nomos-core/risc0_proofs/ptx/src/main.rs

```diff
@@ -0,0 +1,28 @@
+/// Input Proof
```

Comment:
Again, old comment, this is not an input proof, you can simply remove the comment